cli/tests/test_cmd_setup_rotate_token.py

```
@@ -121,6 +121,36 @@ def test_preserves_unrelated_bytes_and_crlf(self) -> None:
             )
 
+    def test_renderer_removes_exact_legacy_key_but_preserves_prefixed_keys(self) -> None:
+        secret = "replacement-token-that-must-stay-redacted"
+        snapshot = cmd_setup._RotateTokenDotenvSnapshot(
+            existed=True,
+            body=(
+                b"OPENCLAW_GATEWAY_TOKEN=exposed-a\r\n"
+                b"OPENCLAW_GATEWAY_TOKEN_SUFFIX=keep-legacy-prefix\r\n"
+                b"XOPENCLAW_GATEWAY_TOKEN=keep-leading-prefix\r\n"
+                b"DEFENSECLAW_GATEWAY_TOKEN=exposed-b\r\n"
+                b"DEFENSE" b"CLAW_GATEWAY_TOKEN_BACKUP=keep-canonical-prefix\r\n"
```

Ruff (0.16.0)

[warning] Implicitly concatenated string literals on one line

Combine string literals

(ISC001)